From e4adbab1f577d9a88b5056c5dad41ae7ee79f977 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 7 Jun 2022 14:00:37 +0100
Subject: [PATCH] ARM: dts: zero-w: Better names for GPIOs 14 & 15

GPIOs 14 and 15 are currently labelled TXD0 and RXD0, which matches
comments on the schematic but 1) doesn't reflect their likely usage
(i.e. UART1) and 2) clashes with GPIOs 32 and 33. Adopt the common
naming for those two pins on BT-equipped Pis - TXD1 and RXD1.

See: https://github.com/raspberrypi/linux/issues/5058

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2708-rpi-zero-w.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
index d96a5c82e069..b49c3cb1438d 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
@@ -45,8 +45,8 @@ &gpio {
 			  "GPIO12",
 			  "GPIO13",
 			  /* Serial port */
-			  "TXD0",
-			  "RXD0",
+			  "TXD1",
+			  "RXD1",
 			  "GPIO16",
 			  "GPIO17",
 			  "GPIO18",
-- 
2.32.0

